helm_chart/recordingbot/values.yaml

```diff
@@ -0,0 +1,18 @@
+# global:
+app:
+  replica: 3
+  image: quay.io/asagtani_wire/recordingbot
+  tag: latest
+  env:
+    - name: SERVICE_TOKEN
+      value: # Provide the Recording bot authentication token here
+    - name: PUBLIC_URL
+      value: http://www.example.com # Link to the subdomain pointing to recordingbot
+    - name: DB_URL
+      value: jdbc:postgresql://data-postgres:5432/recording # Link to your Postgres server
+    - name: DB_USER
+      value: recording # Provide your Postgres DB username
+    - name: DB_PASSWORD
+      value: recording # Provide your Postgres DB password
+    - name: APPENDER_TYPE
```

Comment:
This should probably be hardcoded in the deployment spec. There's no need to make this configurable from the helm chart.